js/start.js

```diff
@@ -54,6 +54,11 @@ app.localSettings.on('change:language', (localSettings, lang) => {
   .open();
 });
 
+app.hiRez = (
```

Comment:
1.) Why is the matchMedia part necessary? Wouldn't chromium always give us the devicePixelRatio?

2.) Instead of attaching this to app, can we make this into a utility? Either make an appropriately named module or (if it's a one-off) add it to utils/index.js.